### ISSUE TITLE
dart version is incompatible

### ISSUE THREAD

**Issue by wuhaozuishuai** [google/json_serializable.dart#771]
Dart 2.12.0-29.10.beta is incompatible with your dependencies' SDK constraints. Please run "pub get" again.

**Comment by kevmoo** [comment]
Which version of json_serialible are you trying to use?

**Comment by wuhaozuishuai** [comment]
3.2.0

**Comment by kevmoo** [comment]
Where are you getting that message output?

Could you include context?

**Comment by wuhaozuishuai** [comment]
i get that when i run"flutter packages pub run json_model"

**Comment by kevmoo** [comment]
Oh...what's `json_model`?
Could you give a dump of the entire output?

**Comment by wuhaozuishuai** [comment]
```  ╔════════════════════════════════════════════════════════════════════════════╗
  ║ A new version of Flutter is available!                                     ║
  ║                                                                            ║
  ║ To update to the latest version, run "flutter upgrade".                    ║
  ╚════════════════════════════════════════════════════════════════════════════╝


Error: No pubspec.yaml file found.
This command should be run from the root of your Flutter project.```

sorry ,I am a beginner.

**Comment by wuhaozuishuai** [comment]
but my `flutter` is up to date

**Comment by kevmoo** [comment]
You have to run it in the directory with `pubspec.yaml`

**Comment by wuhaozuishuai** [comment]
I also found this problem,let me try again.thanks

**Comment by wuhaozuishuai** [comment]
i find a new question
`Dart 2.12.0-29.10.beta is incompatible with your dependencies' SDK constraints. Please run "pub get" again.
pub finished with exit code 65
`
i 've run pub get but trashy

**Comment by BenjaminFarquhar** [comment]
@wuhaozuishuai does your `pubspec.yaml` contain any `dependency_overrides:` or `path:`? I was getting an `exit code 65` due to that having that whilst having `flutter_gen:`. I had to change `flutter_gen:` to `flutter_gen_runner:`.
